horizon: 20
corridorConstrains: 1
qVTheta: 0.005
# 2 order:
# stateUpper: [ 100,  3,  100,  3,  100,  3,  100,  100]
# stateLower: [-100, -3, -100, -3, -100, -3, -0.1, -0.1]
# 3 order:
stateUpper: [ 100,  10,  10,  100,  10,  10,  100,  10,  10,  100,     1,  20]
stateLower: [-100, -10, -10, -100, -10, -10, -100, -10, -10, -0.1,  -0.1, -20]
inputUpper: [  8,  8,  8,  15]
inputLower: [ -8, -8, -8, -15]


# horizon: 30
# corridorConstrains: 1
# qVTheta: 0.2
# # 2 order:
# # stateUpper: [ 100,  3,  100,  3,  100,  3,  100,  100]
# # stateLower: [-100, -3, -100, -3, -100, -3, -0.1, -0.1]
# # 3 order:
# stateUpper: [ 100,  10,  10,  100,  10,  10,  100,  10,  10,  100,   15,  15]
# stateLower: [-100, -10, -10, -100, -10, -10, -100, -10, -5, -0.1,  -0.1, -15]
# inputUpper: [  10,  10,  10,  10]
# inputLower: [ -10, -10, -10, -10]